lib/rubocop/cop/layout/heredoc_argument_closing_parenthesis.rb

```diff
@@ -108,6 +108,10 @@ def autocorrect(node)
           end
         end
 
+        def self.autocorrect_incompatible_with
+          [RuboCop::Cop::Style::TrailingCommaInArguments]
```

Comment:
It seems the convention across the code base is:
```suggestion
          [Style::TrailingCommaInArguments]
```

Reply:
I think there should be an inverse definition inside `Style/TrailingCommandInArguments.autocorrect_incompatible_with`.